src/storages/user.storage.js

```diff
@@ -14,35 +16,63 @@ const createUser = (req, res) => {
     }
 
     // saved!
-    res.status(200).json({ message: "User cerated!", doc });
+    res.status(200).json({ message: "User created!", doc });
   });
 };
 
+//an ADMINISTRATOR can obtain a user from its id
```

Comment:
Why *ADMINISTRATOR*? A normal user cannot obtain info about his personal account?

Reply:
L'operazione di un utente di visualizzare le sue informazioni sul suo account personale c'è ed è in authStorage (/api/profile per intenderci). In userstorage avevo messo delle operazioni che potrebbe fare un amministratore, tipo visualizzare un utente dato il suo id o visualizzare la lista degli utenti registrati. Questa parte se è ritenuta inutile si può togliere